Autopilot/TelemetryManager/telemetryManager.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Around here,we sometimes like to write our names down as Authors at the top of the files we write. Feel free to do that if you like!